SignalServiceKit/src/Messages/OWSMessageManager.m

```diff
@@ -313,19 +314,19 @@ - (void)handleEnvelope:(OWSSignalServiceProtosEnvelope *)envelope
         } else {
             DDLogWarn(@"%@ Ignoring envelope. Content with no known payload", self.logTag);
         }
-    } else if (envelope.hasLegacyMessage) { // DEPRECATED - Remove after all clients have been upgraded.
+    } else if (envelope.legacyMessage != nil) { // DEPRECATED - Remove after all clients have been upgraded.
         OWSSignalServiceProtosDataMessage *dataMessage =
             [OWSSignalServiceProtosDataMessage parseFromData:plaintextData];
         DDLogInfo(
             @"%@ handling message: <DataMessage: %@ />", self.logTag, [self descriptionForDataMessage:dataMessage]);
 
         [self handleIncomingEnvelope:envelope withDataMessage:dataMessage transaction:transaction];
     } else {
-        OWSProdInfoWEnvelope([OWSAnalyticsEvents messageManagerErrorEnvelopeNoActionablePayload], envelope);
+        DDLogWarn(@"%@ Ignoring envelope with neither Content nor LegacyMessage", self.logTag);
```

Comment:
`OWSProdInfoWEnvelope()` was a placeholder for any eventual analytics.  I think this is the only place we're using it, so if you want to remove it, we should remove that macro.  But maybe put in a TODO: here to add analytics?

Reply:
Sorry - this was unintentional. I've reverted this change.